core/runtime/common/runtime_instances_pool.hpp

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright Soramitsu Co., Ltd. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0
- */
+* Copyright Soramitsu Co., Ltd. All Rights Reserved.
```

Comment:
Please revert this file (it's a fix from master)

Reply:
It will reverted automatically when temporary merged branch will be removed by rebase after merge that branch to master